### PR TITLE
It's dangerous out there

### PR DESCRIPTION
**Pull request by cleborys**
I thought that it would be fun to have more factions, temporary alliances or betrayals going on, so I wanted to propose that players who are alone on a field of the board receive some form of punishment. 
Current ideas are loss of some points (boring) or being sucked one step closer to the black hole (less boring).

### CODE REVIEW

**Comment by cleborys** [comment]
Discussion should be moved here.

**Comment by Snark1994** [comment]
master branch has been fixed.

**Comment by cleborys** [comment]
I agree that with 4 Players this will probably settle into two 2-Player factions quickly.
I'll think of some ways to mix tis up, like playing hide-and-seek or some version of the prisoners dilemma. Suggestions welcome.

**Comment by Snark1994** [comment]
Bidding to move the minotaur? E.g. everyone is allowed to move the minotaur as well as their king, majority decision (if any) decides which way it goes; can optionally also pay some points, and if so then  it's the majority decision out of those who paid the maximal number of points

**Comment by cleborys** [comment]
Current idea is this: If all but a single player are grouped, that player will gain points quadratically fast (up to a cap), but if there are several isolated players, they will all lose points but will simultaneously make it less rewarding to be a unique isolated player.
Hopefully, this sets up an environment, whose equilibrium oscillates between a single player defecting and all players spreading out to reset the stakes.
I tried to make this not too dangerous of causing someone to win the game by introducing a cap on point growth, but still keeping some of the sting.

As this needs the option to regroup quickly in order to properly work, there is an additional "teleport"-option.

**Comment by cleborys** [comment]
possible adjustment: 
if there are two or more isolated players, these lose points, but the bounty only decreases if all players are isolated.

**Comment by malte-l** [comment]
Some quick comments:

* there are some spelling mistakes: "tto", "groupoing".
* "In order to facilitate groupoing on the board": I assume this is just reasoning for the following option to exist, not a requirement to using it (i.e. a player must somehow prove that they are just teleporting in order to facilitate grouping on the board and not some other reason). In that case, something like this belongs in commentary to the rule, but not the rule itself and I would recommend removing it.
* If player X sends a command to teleport to player Y, and player Y sends a command to teleport to player X, what happens? I suggest to specify the order in which such commands are evaluated. For example in the order the players move, starting from the player after the player whose turn it was when the commands were sent.

**Comment by malte-l** [comment]
Ah, I just saw your earlier commit, one really has to be fast here to not be immediately out of date ;)
"tto" is in the second sentence: "[...] points that would cause any player tto win the game [...]".

**Comment by cleborys** [comment]
Not sure why CTRL+F didnt find that.

Any feed back on whether this is fun / too dangerous?

File: rules.md [hunk 1]
```diff
@@ -124,3 +124,10 @@ This rule takes precedence over every other rule determining the winner except r
...
     - Unless specified differently in other rules each player is only allowed to move his own king. Kings of different players do not interact. Neither do kings and items or items and items interact with each other unless this is specified explicitly in another rule (and apart from the black hole discribed in this rule).
     - The state of the board can only change if this is explicitly allowed by a rule.
+* 306: There shall be a variable BOUNTY whose initial value when this rule passes is 10 that may be kept track of in ´scores.md´. This variable shall always take values of an integer greater or equal to 1 but not exceeding a third of any amount of points that would cause any player to win the game, if such an amount is specified by any other rule. This range will be called "legal values" below.
```
Comment (Snark1994) [inline rules.md]: I think this is confusingly phrased - could you just say "no more than a third of the point total specified in rule _N_", and we can be careful not to mess it up in future?

File: rules.md [hunk 2]
```diff
@@ -124,3 +124,10 @@ This rule takes precedence over every other rule determining the winner except r
...
+Whenever a players king, after executing the move comands as by rule 305 on voting days, is alone on a field (called "isolated" below), the player shall immediately gain or lose points as specified below:
```
Comment (Snark1994) [inline rules.md]: "player's king"

File: rules.md [hunk 3]
```diff
@@ -124,3 +124,10 @@ This rule takes precedence over every other rule determining the winner except r
...
+    - If there is no isolated player, no player receives points accoding to this rule.
```
Comment (Snark1994) [inline rules.md]: "according"

File: rules.md [hunk 4]
```diff
@@ -124,3 +124,10 @@ This rule takes precedence over every other rule determining the winner except r
...
+    - If there is a single isolated, that player receives the number of points specified in the variable BOUNTY. Consequently, BOUNTY is increased by 10 and then adjusted down to the closest legal value, if necessary.
```
Comment (Snark1994) [inline rules.md]: "single isolated player,"

File: rules.md [hunk 5]
```diff
@@ -124,3 +124,10 @@ This rule takes precedence over every other rule determining the winner except r
...
+    - If there are two or more isolated players, all isolated players lose the number of points specified in BOUNTY. Consequently, BOUNTY is halved and rounded down to an integer, then adjusted up to the closest legal value, if necessary.
+Any player who is isolated before executing a move command as by rule 305 may issue a "TELEPORT X" command additionally to issuing a move direction as in rule 305, where X is the identifier of a player as in ´scores.md´. This causes the players king to be moved to the field that the king of player X is on, before any non-teleport move is executed. The move command accompanying teleporting may be empty. If there are several teleport commands, these are executed starting at the player whose turn it currently is and then traversing in reverse order of play.
```
Comment (Snark1994) [inline rules.md]: "player's king" and "moved to the square that the king of"

**Comment by Snark1994** [comment]
It seems interesting, I worry that it will lead to nobody wanting to move though... I like the idea of the bounty that fluctuates in value, though. (Sorry it's late in the day, I have had Danish all evening.)

**Comment by Snark1994** [comment]
Perhaps the same idea could be achieved with items? _e.g._ introducing _N_ identical items that you are allowed to give (or steal?), and you get points if you are the only player with precisely one of said items. (For _N_ an appropriately chosen small number.) That way we keep the prisoner's dilemma without encouraging people not to be moving, which mechanic I would quite like to keep usable.

**Comment by cleborys** [comment]
It's not late in the day for me ;)
I don't think this discourages moving. Actually you even **have** to move to prevent a single dissident from winning. It does of course restrict your ability to move freely somewhat by mixing in other incentives, but this is what I liked (rather than having a set of parallel but independent minigames)

**Comment by AnonymousVeNomicPlayer** [comment]
Just to let you now, I read the proposal quickly and do not have any more comments on first sight

**Comment by Snark1994** [comment]
It occurs to me that this gives the proposer of the rule tremendous power: they know exactly where everyone will be when they decide what their move was, and so can ensure they're isolated iff they want to be.

**Comment by cleborys** [comment]
I had noticed that, but thought it's not too big of a deal, since we are taking turns proposing rules and the decision is not completely free: while you can freely choose whether you are isolated or not, you can't freely chose whether you are the unique isolated player and that means that sometimes, as proposer, you will need to decide between you and X losing some points or X gaining an equal amount of points. For the most part this means that if no-one else choses to separate from the group, the proposer will.

**Comment by cleborys** [comment]
Anyways, the rule passed 3 to 1,
M moved up and P moved right.

To avoid another judgement: Do M and P lose 10 points each now? That is, did the last judgement rest on commands being sent before the rule passed or on the rule being in effect after the turn ended?

**Comment by AnonymousVeNomicPlayer** [comment]
Who voted no? (According to the rules you must state how each player voted so that they are able to check that their vote was counted correctly.)
Also you should update the scores.

**Comment by Snark1994** [comment]
A gentle reminder to update scores when you have time!

**Comment by cleborys** [comment]
Didn't see P's request, sorry. J voted no.

**Pull request merged**